src/index.html

```diff
@@ -8,7 +8,224 @@
     <link rel="stylesheet" href="./css/index.scss" />
   </head>
   <body>
-    <div id="main"></div>
+    <div id="main">
+      <!--Header-->
+      <header>
+        <div>
+          <span>Home</span>
+          <span style="text-align: right;">Download My Full Resume</span>
+
+        </div>
+      </header>
+
+      <body>
```

Comment:
`<body></body>` should only appear once and wrap all the body content. Please remove this instance here.